Makefile

```diff
@@ -30,12 +30,12 @@
   test-boot-py\
   test-boot-ocaml\
   test-sundials\
-	test-ipopt\
+  test-ipopt\
   test-par
 
 all: build
 
-boot:
+boot: lint
```

Comment:
Having `lint` as a dependency for `boot` will force all users to run it, won't it? Shouldn't it be on `test-boot-base` instead?

Reply:
For some reason, it is needed as a prereq of `boot` specifically. If I put it on `test-boot-base` instead, it gives the error. I think the issue is building multiple dune targets simultaneously.